resources/js/components/StartExperimentBtn.vue

```diff
@@ -20,7 +20,9 @@
 			    </div>
 			  </div>
 		</div>
-		
+		<span v-if="timeexpires==true" class="start" style="position: absolute;top:45%; left: 45%;  cursor: pointer;background: #a23;color: #fff;padding: 15px 20px;border-radius: 8px;">
+			<span class="fa fa-lock mr-1" ></span>Time Up
+		</span>
 		<span class="start" style="position: absolute;top:78%; left: 45%;  cursor: pointer;background: #7d9;color: #fff;padding: 15px 20px;border-radius: 8px;" @click="startT" v-if="start==false">
```

Comment:
I see you have a start class here, why are we having inline styling then? why not add an additional class for this still if you cant add the styling to the .start properties